### DISCUSSION TITLE
Hi i need some help to get started.

### DISCUSSION THREAD

**Discussion by holymoses1** [microsoft/winget-cli#2579] [General]
I need a function that shows a list on the home page of clans, who is in the clan, etc.
And which allows the person who created the clan to edit it himself.

- If I create a quad clan, and add 3 others to that clan, that clan would not appear on the list until the other 3 had accepted a "request" to become a member of that clan.
The usernames of the members of the clan would be hyperlinks to the members' profile page.

- If one member no longer wants to be part of the clan, the member must have the opportunity to delete himself from the clan. Then, by definition, the clan is no longer complete, and the system must delete the clan from the list.

- The list/form must update live to a widget that I can place on a suitable page.

- The list must be able to be sorted in the widget based on clan name and member name

**Comment by denelon** [comment]
@holymoses1 I'm not sure how this applies to the Windows Package Manager.
